### PR TITLE
LRCI-2556 add batches for db2 11.5

### PR DESCRIPTION
**Pull request by CharlotteFrancis**
https://issues.liferay.com/browse/LRCI-2556

### CODE REVIEW

**Comment by CharlotteFrancis** [comment]
ci:test

**Comment by pyoo47** [comment]
ci:stop

**Comment by pyoo47** [comment]
ci:test:sf

**Comment by pyoo47** [comment]
ci:test:relevant

**Comment by pyoo47** [comment]
ci:test:relevant

**Comment by pyoo47** [comment]
ci:test:relevant

File: test.properties [hunk 1]
```diff
@@ -4666,6 +4740,83 @@
 
     test.release.bundle[portal-release-tomcat]=true
 
+    #
+<<<<<<< HEAD
```
Comment (pyoo47) [inline test.properties:4744]: @CharlotteFrancis It looks like you somehow overrode a merge-conflict and commited this file. Can you fix the merge conflict and resend?

**Pull request closed without merging**